spec/realworld/parallel_spec.rb

```diff
@@ -9,7 +9,9 @@
       gem 'i18n', '~> 0.6.0' # Because 0.7+ requires Ruby 1.9.3+
     G
 
-    bundle :install, :jobs => 4, :env => { "DEBUG" => "1" }
+    bundle "config jobs 4"
+    # NOTE: The env option below cannot be configured with `bundle "config env debug"`
+    bundle :install, :env => { "DEBUG" => "1" }
```

Comment:
it's actually possible to just use `bundle install --verbose` here (verbose output contains the thread numbers, which is why we use verbose output in this test)